src/Components/Map.tsx

```diff
@@ -8,9 +8,6 @@ import { SearchContext } from "../App";
 import { MAPBOX_TOKEN } from "../constants";
 import { getFilteredPins } from "../utils/getFilteredPins";
 
-// single value for now but should change exponentially
-const getDelta = (zoom: number): number => 0.05 / zoom;
```

Comment:
just making a note that this is gone. verified that this was intentional via slack